src/containers/profile/ProfileSections/AccountSettings/AccountSettingsSections/ProfileInfo/ProfileInfo.css

```diff
@@ -0,0 +1,16 @@
+.editButton {
```

Comment:
```.editButton ``` is not a descriptive class name. it could be an edit button for profile or blogs or ...etc

Reply:
yes you're right